etc/travis/test-images-dockerhub.sh

```diff
@@ -0,0 +1,11 @@
+# fail fast
+set -e
+export MAKE_ARGS=--no-print-directory
+# pull images from dockerhub
+make $MAKE_ARGS pull-dockerhub-images
+make $MAKE_ARGS tag-dockerhub-images-to-latest
```

Comment:
Looks like a good workaround solution for now, but I guess at some point we should make the Docker tags parameterizable in the deployment script, so we don't need to overwrite `latest` tag here.